src/Audit/Formatters/TextFormatter.ts

```diff
@@ -71,13 +71,20 @@ export class TextFormatter implements Formatter {
       x.forEach((y: Vulnerability) => {
         const color: string = this.getColorFromMaxScore(+y.cvssScore);
         console.group();
-        console.log(chalk.keyword(color)(`Vulnerability Title: `), `${y.title}`);
-        console.log(chalk.keyword(color)(`ID: `), `${y.id}`);
-        console.log(chalk.keyword(color)(`Description: `), `${y.description}`);
-        console.log(chalk.keyword(color)(`CVSS Score: `), `${y.cvssScore}`);
-        console.log(chalk.keyword(color)(`CVSS Vector: `), `${y.cvssVector}`);
-        console.log(chalk.keyword(color)(`CVE: `), `${y.cve}`);
-        console.log(chalk.keyword(color)(`Reference: `), `${y.reference}`);
+        if (typeof y.title !== 'undefined') 
```

Comment:
You don't need to do this in this PR, but you could write something like `doPrint(thing: string): void` that would be more or less like:

```
private doPrint(color: string, title: string, text: string): void {
        if (typeof text !== 'undefined')
          console.log(chalk.keyword(color)(title), text);
}
```